lightning/backend/allocator.go

```diff
@@ -19,33 +19,43 @@ import (
 	"github.com/pingcap/tidb/meta/autoid"
 )
 
-// PanickingAllocator is an ID allocator which panics on all operations except Rebase
-type PanickingAllocator struct {
+// panickingAllocator is an ID allocator which panics on all operations except Rebase
+type panickingAllocator struct {
 	autoid.Allocator
-	base int64
+	base *int64
```

Comment:
em, why change to a pointer?

Reply:
So that all 3 allocators (row_id, auto_inc, auto_random) share the same value when we Rebase